lang/cy.js

```diff
@@ -78,7 +78,7 @@ export default {
 	"table:column:pasteBefore": "Gludo Colofn Cyn",
 	"table:delete": "Dileu Tabl",
 	"table:insert": "Mewnosod Tabl",
-	"table:insert:columsByRows": "{columns, plural, one { 1 golofn } other { { o golofnau } columns }} gan {rows, plural, one { 1 rhes} other { { rows } o resi }}",
+	"table:insert:columsByRows": "{columns, plural, one { 1 golofn } other { { columns } columns }} gan {rows, plural, one { 1 rhes} other { { rows } o resi }}",
```

Comment:
I think some of these still end up not quite being correct, since now they're not translated at all. I think this should probably be:
```suggestion
	"table:insert:columsByRows": "{columns, plural, one { 1 golofn } other { { columns } o golofnau }} gan {rows, plural, one { 1 rhes} other { { rows } o resi }}",
```

Reply:
Good catch!